packages/flutter/lib/src/material/text_field.dart

```diff
@@ -619,6 +620,9 @@ class TextField extends StatefulWidget {
   ///    [TextInputAction.previous] for [textInputAction].
   final ValueChanged<String> onSubmitted;
 
+  /// {@macro flutter.widgets.editableText.onAppPrivateCommand}
```

Comment:
Dartdoc is complaining that there's no template for this macro: https://cirrus-ci.com/task/5126735421243392?command=main#L119

Reply:
Added template in editable_text.dart